mapping_files/material_types.tsv

```diff
@@ -1,188 +1,185 @@
-NONCIRC MAP HOOVER	map
-NONCIRC MARC HOOVER	book
-NONCIRC MRDF HOOVER	electronic resource
-NONCIRC RECORDING HOOVER	multimedia
-NONCIRC SCORE HOOVER	score
-NONCIRC SERIAL HOOVER	periodical
-NONCIRC VM HOOVER	book
-REF MANUSCRPT HOOVER	archival
-REF MAP HOOVER	map
-REF MARC HOOVER	book
-REF MRDF HOOVER	electronic resource
-REF RECORDING HOOVER	multimedia
-REF SCORE HOOVER	score
-REF SERIAL HOOVER	periodical
-REF VM HOOVER	book
-STKS MANUSCRPT HOOVER	archival
-STKS MAP HOOVER	map
-STKS MARC HOOVER	book
-STKS MRDF HOOVER	electronic resource
-STKS RECORDING HOOVER	multimedia
-STKS SCORE HOOVER	score
-STKS SERIAL HOOVER	periodical
-STKS VM HOOVER	book
-VAULT MANUSCRPT HOOVER	archival
-VAULT MAP HOOVER	map
-VAULT MARC HOOVER	book
-VAULT MRDF HOOVER	electronic resource
-VAULT RECORDING HOOVER	multimedia
-VAULT SCORE HOOVER	score
-VAULT SERIAL HOOVER	periodical
-VAULT VM HOOVER	book
-LIBUSEONLY MANUSCRPT HV-ARCHIVE	archival
-LIBUSEONLY MAP HV-ARCHIVE	map
-LIBUSEONLY MARC HV-ARCHIVE	book
-LIBUSEONLY MRDF HV-ARCHIVE	electronic resource
-LIBUSEONLY RECORDING HV-ARCHIVE	multimedia
-LIBUSEONLY SCORE HV-ARCHIVE	score
-LIBUSEONLY SERIAL HV-ARCHIVE	periodical
-LIBUSEONLY VM HV-ARCHIVE	book
-NONCIRC MANUSCRPT HV-ARCHIVE	archival
-NONCIRC MAP HV-ARCHIVE	map
-NONCIRC MARC HV-ARCHIVE	book
-NONCIRC MRDF HV-ARCHIVE	electronic resource
-NONCIRC RECORDING HV-ARCHIVE	multimedia
-NONCIRC SCORE HV-ARCHIVE	score
-NONCIRC SERIAL HV-ARCHIVE	periodical
-NONCIRC VM HV-ARCHIVE	book
-STKS MANUSCRPT HV-ARCHIVE	archival
-STKS MAP HV-ARCHIVE	map
-STKS MARC HV-ARCHIVE	book
-STKS MRDF HV-ARCHIVE	electronic resource
-STKS RECORDING HV-ARCHIVE	multimedia
-STKS SCORE HV-ARCHIVE	score
-STKS SERIAL HV-ARCHIVE	periodical
-STKS VM HV-ARCHIVE	book
-VAULT MANUSCRPT HV-ARCHIVE	archival
-VAULT MAP HV-ARCHIVE	map
-VAULT MARC HV-ARCHIVE	book
-VAULT MRDF HV-ARCHIVE	electronic resource
-VAULT RECORDING HV-ARCHIVE	multimedia
-VAULT SCORE HV-ARCHIVE	score
-VAULT SERIAL HV-ARCHIVE	periodical
-VAULT VM HV-ARCHIVE	book
-LIBUSEONLY MANUSCRPT	archival
-LIBUSEONLY MAP	map
-LIBUSEONLY MARC	book
-LIBUSEONLY MARC GREEN	microform
-LIBUSEONLY SERIAL	periodical
-LIBUSEONLY VM	multimedia
-NH-7DAY SCORE	score
-NH-7DAY SERIAL	periodical
-NH-7DAY RECORDING	multimedia
-NH-7DAY VM	multimedia
-NH-7DAY MAP	map
-NH-7DAY MRDF	multimedia
-NH-7DAY MARC	book
-NH-MEDSTKS SCORE	score
-NH-MEDSTKS SERIAL	periodical
-NH-MEDSTKS RECORDING	multimedia
-NH-MEDSTKS MAP	map
-NH-MEDSTKS MRDF	multimedia
-NH-MEDSTKS MARC	book
-NH-MEDSTKS VM	multimedia
-NH-INHOUSE MANUSCRPT	archival
-NH-INHOUSE MAP	map
-NH-INHOUSE MARC	book
-NH-INHOUSE MRDF	multimedia
-NH-INHOUSE RECORDING	multimedia
-NH-INHOUSE SCORE	score
-NH-INHOUSE SERIAL	periodical
-NH-INHOUSE TECHRPTS	book
-NH-INHOUSE VM	multimedia
+ITEM_TYPE	FORMAT	LIBRARY	folio_name
+*	*	*	book
+ARCHIVE	*	*	archival
+ARCHIVEREF	*	*	archival
+ATLAS	*	*	map
+ATLAS-EXP	*	*	map
+AUDIO	*	*	multimedia
+CAREERCOLL	*	*	book
+CJK	*	*	book
+CLUSTER	*	*	electronic resource
+DATABASE	*	*	electronic resource
```

Comment:
Are we no longer using software and database material types?

Reply:
material types are still used but for circulation purposes, these are mapped to electronic resource.